zk-sdk/src/sigma_proofs/percentage_with_cap.rs

```diff
@@ -10,12 +10,12 @@
 //! max cap value.
 //!
 //! A more detailed description of the context and how the proof is computed is provided in the
-//! [`ZK Token proof program`] documentation, specifically the percentage-with-cap [`specification`].
+//! [`ZK ElGamal proof program`] documentation, specifically the percentage-with-cap [`specification`].
 //!
 //! The protocol guarantees computational soundness (by the hardness of discrete log) and perfect
 //! zero-knowledge in the random oracle model.
 //!
-//! [`ZK Token proof program`]: https://docs.solanalabs.com/runtime/zk-token-proof
+//! [`ZK ElGamal proof program`]: https://edge.docs.solanalabs.com/runtime/zk-token-proof
```

Comment:
This link doesn't seem right, how about this
```suggestion
//! [`ZK ElGamal proof program`]: https://docs.anza.xyz/runtime/zk-elgamal-proof
```

Reply:
Oh oops. I should have caught this one... thanks for the catch!